web/public/mgmt/config.yml

```diff
@@ -1458,6 +1458,13 @@ collections:
         default: ""
         required: false
 
+      - label: "Municipality"
+        name: "municipality"
+        widget: "select"
+        multiple: false
+        required: false
+        options: ["All", "Newark"]
```

Comment:
It may be better long term to have this pulling from a file but for now we only have one municipality we are tuning into. Open to pulling it out if folks prefer though

Reply:
I am okay with this approach.

Reply:
Same